platform/jobs/RetirementJobEdxTriggers.groovy

```diff
@@ -0,0 +1,91 @@
+/* UserRetirementJobEdxTriggers.groovy
+ *
+ * Defines jobs which trigger downstream user retirement jobs for each edX
+ * environment.
+ */
+
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_TEAM_SECURITY
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+List jobConfigs = [
+    [
+        environmentDeployment: 'prod-edx'
+    ],
+    [
+        environmentDeployment: 'stage-edx'
+    ],
+    [
+        environmentDeployment: 'loadtest-edx'
+    ],
+    [
+        environmentDeployment: 'prod-edge'
+    ]
+]
+
+
+jobConfigs.each { jobConfig ->
+
+    // ########### user-retirement-trigger-<environment> ###########
+    // This defines the job which triggers the collector job for a given environment.
+    job("user-retirement-trigger-${jobConfig.environmentDeployment}") {
+        description("Scheduled trigger of the user-retirement-collector job for the ${jobConfig.environmentDeployment} environment")
+
+        // Only a subset of edx employees should be allowed to control this job,
+        // but customer support can read and discover.
+        authorization {
+            blocksInheritance(true)
+            List membersWithFullControl = ['edx/platform-team']
+            membersWithFullControl.each { emp ->
+                permissionAll(emp)
+            }
+            // TODO PLAT-2036: uncomment the following two lines when we add the
+            // appropriate github group.
+            //permission('hudson.model.Item.Read', 'edx/customer-support')
+            //permission('hudson.model.Item.Discover', 'edx/customer-support')
+        }
+
+        // backup-runner is the jenkins master, for us, which is fine because
+        // this job only triggers downstream jobs.
+        label('backup-runner')
```

Comment:
it occurs to me that the default configuration for build jenkins master instance only has 1 executor, so this job would probably tie up the master for too long.  I'll switch the label for this job back to a jenkins-worker and put in ticketing to figure out the workers situation.